sigal/video.py

```diff
@@ -61,10 +61,14 @@ def check_subprocess(cmd, source, outname):
         raise SubprocessException('Failed to process ' + source)
 
 
-def video_size(source):
-    """Returns the dimensions of the video."""
+def video_size(source, settings={}):
```

Comment:
I would use directly a `video_converter` parameter (or `converter`) here to avoid having to pass a dict just for one key.

Reply:
Also do not create a dict in the function definition, because if you do not provide different settings that same dict will be used each time, potentially causing problems.

From flake8-bugbear:
> B008: Do not perform calls in argument defaults. The call is performed only once at function definition time. All calls to your function will reuse the result of that definition-time call. If this is intended, assign the function call to a module-level variable and use that variable as a default value.